<?xml version="1.0"?>
<launch>
	<node pkg="visp_auto_tracker" type="visp_auto_tracker" name="visp_auto_tracker" output="screen">
	    <param name="model_path" value="$(find visp_auto_tracker)/models" />
	    <param name="model_name" value="pattern" />
	    <param name="debug_display" value="true" />
	    <!-- param name="code_message" value="Ver1" / -->
	    
	    <remap from="/visp_auto_tracker/camera_info" to="/kinect2/qhd/camera_info"/>
	    <remap from="/visp_auto_tracker/image_raw" to="/kinect2/qhd/image_color_rect"/>
      </node>

</launch>
